server/l10n/messages.pot

```diff
@@ -207,6 +190,41 @@ msgstr ""
 msgid "Gross National Income"
 msgstr ""
 
+#. Blog post entry. Please take care to maintain valid <a> tags in translation. See this string in action at https://datacommons.org
+#: server/templates/static/homepage.html:56
+msgid "HOME-BLOG_POST_1"
+msgstr "%(project)s is now accessible on Google Search! <a href=%(blog_url)s>Read more here</a>"
+
+#. Describes building your own tools and analysis using our API (here Python refers to the programming language - see python.org, and REST is an acronym for REST is an acronym for REpresentational State Transfer). Please see this string in action at https://datacommons.org
```

Comment:
remove

Reply:
Sorry, what needs to be removed? This string is needed for the homepage. The translators also want very detailed descriptions, so it's easier to just point them to the homepage so they can see the context.

Reply:
this sentence: "and REST is an acronym for REST is an acronym for REpresentational State Transfer)"

Reply:
wow, thanks for that catch! i really couldn't see that.